docs/informatiemodel/semantisch_model_oud.md

```diff
@@ -10,6 +10,23 @@ date: 29-09-2023
 Binnen VNG Realisatie wordt Enterprise Architect (EA) gebruikt voor de vastlegging van semantische informatiemodellen (SIM). 
 Onderstaand zijn de diagrammen van de oude versies van het infromatiemodel weergegeven en zijn de wijzigingen opgesomd die in het nieuwe model zijn doorgevoerd. 
 
+## Van versie 0.0.3 naar 0.0.4 (04-10-2023)
+Onderstaand figuur geeft de modellering van het SIM versie 0.0.3 weer zoals deze in EA was gemodelleerd.
+
+<img src="assets/SIM_Klantinteracties_v003.png" alt="SIM in Enterprise Architect" width="1000"/>
+
+De wijzigingen die in versie 0.0.4 zijn doorgevoerd t.o.v. deze versie zijn: 
+  - Primitief datatype Tkest (1000 karakters) toegevoegd
+  -	Objecttype Klantontact:
+    -	Gemuteerd op toegevoegd.
```

Comment:
Bij nader inzien is het volgens mij de vraag of deze attributen horen bij het klantcontact zelf of een versie daarvan. Ik denk het laatste. Dit betekent praktisch gezien dat het mutatiemoment en de toelichting daarbij thuishoren in het auditlog, en niet in het semantisch model.

Reply:
Wil je van de meest actuele versie dan niet weten wat de mutatiedatum / reden is?  Of is dat niet relevant voor ondersteuning van het proces en alleen relevant om na te kunnen kijken vanuit een audit-perspectief ? 

In het tweede geval: Wij modelleren het Audit log niet en dan zouden deze eigenschappen dus verwijderd moeten worden uit het informatiemodel.... Mee eens ?

Reply:
Aanpassingen doorgevoerd zoals vanmorgen besproken.